src/sentry/api/endpoints/user_password.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import absolute_import
+
+from django.utils.crypto import constant_time_compare
+from rest_framework import serializers, status
+from rest_framework.response import Response
+
+from sentry.api.bases.user import UserEndpoint
+from sentry.api.decorators import sudo_required
+from sentry.auth import password_validation
+from sentry.models import User
+from sentry.security import capture_security_activity
+
+
+class UserPasswordSerializer(serializers.ModelSerializer):
+    passwordVerify = serializers.CharField(max_length=128, required=True)
+    passwordNew = serializers.CharField(max_length=128, required=True)
```

Comment:
Is there a reason we're limiting to 128 characters here? That's the only real input here. I don't think there was any limit before, and I don't feel strongly that there should be since there's no real reason not to allow longer since we just store a hash, and computing the hash cost isn't significant enough. (This has come up before and I ruled the cost irrelevant. It wasn't possible to POST a large enough payload that would have an impact.)

Reply:
Yeah the 128 char limit seems like a mistake, will fix, thanks